lib/wallet/src/Cardano/Wallet/Write/ProtocolParameters.hs

```diff
@@ -23,28 +22,25 @@ import qualified Cardano.Wallet.Write.Tx as WriteTx
 -- TODO:
 --  - Make this data type abstract: don't export the constructor.
 --  - Replace this type with a re-exported 'Ledger.PParams era'.
-data ProtocolParameters era = ProtocolParameters
-    { pparamsWallet
-        :: !Wallet.ProtocolParameters
-    , pparamsLedger
-        :: !(WriteTx.PParams (WriteTx.ShelleyLedgerEra era))
+newtype ProtocolParameters era = ProtocolParameters
```

Comment:
Leaving it as is for now. I'd like to revisit `unsafeFromWalletProtocolParameters` and this type and module when returning to #3874.

Reply:
I agree it's fine to leave as it is for now. 👍🏻